syntax="proto3";
package game;

message ReqLogin {
  int32 msg_id = 1;
  string account_id = 2;
  int64 device_id = 3;
}

message ResLogin {
  int32 msg_id = 1;
  int64 actor_id = 2;
}